al2023-support-statements/README.md

```diff
@@ -0,0 +1,143 @@
+## Amazon Linux 2023 Support Statements
+
+Just want to see the support statements? Head to https://docs.aws.amazon.com/linux/al2023/release-notes/support-info-by-support-statement.html
+
+This repository contains information as to the Maintenance Support period of Amazon Linux 2023.
+
+The information is available in a number of formats. There is the machine readable `support_info.xml`, which is of the `support_info.xsd` schema. For humans, there is XSLT to have a by package and by support statement ordered output.
+
+In Amazon Linux 2023, you can see specific support status and dates of individual packages using the `dnf-plugin-support-info` plugin. You can even get information on all currently installed packages or apply filters based on "supported" or "unsupported" packages.
+
+DNF Plugin source is available here: https://github.com/amazonlinux/dnf-plugin-support-info.
+
+
+## DNF SupportInfo Pugin
+
+Amazon Linux provides updates for all packages and maintains compatibility within a major version for customer applications that are built on Amazon Linux. Core packages, such as glibc, openssl, openssh, and the dnf package manager, receive support for the lifetime of the major Amazon Linux 2023 release. Packages that aren't part of the core packages receive support defined by their upstream sources.
+
+The full list of core packages will be ﬁnalized during the preview. If you want to see more packages included as core packages, tell us. We will evaluate as we are collecting feedback. Feedback on Amazon Linux 2023 can be provided through your designated AWS representative or [Amazon Linux Discussion Forums](https://forums.aws.amazon.com/forum.jspa?forumID=228).
```

Comment:
This paragraph should be deleted. We're not in preview anymore, and the forums are dead.